native/src/components/base/TextButton.tsx

```diff
@@ -25,12 +25,31 @@ type TextButtonProps = {
   text: string
   onPress: () => Promise<void> | void
   disabled?: boolean
+  accessible?: boolean
+  focusable?: boolean
+  accessibilityRole?: AccessibilityRole | undefined
```

Comment:
No need to have the `undefined` here, as this prop is optional, because of the question mark, the `undefined` type is added automatically.

Also there is no need to have the `accessibilityRole` as a prop here, as we are in a Button component, the accessibilityRole should always be "button" here. So just add below `accessibilityRole='button'` and remove the option to pass an other prop.